src/include/sof/audio/coefficients/fft/twiddle_32.h

```diff
@@ -1,29 +1,29 @@
 /* SPDX-License-Identifier: BSD-3-Clause
  *
- * Copyright(c) 2020 Intel Corporation. All rights reserved.
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
  *
  */
 
 /* Twiddle factors in Q1.31 format */
 
-#ifndef __INCLUDE_TWIDDLE_H__
-#define __INCLUDE_TWIDDLE_H__
+#ifndef __INCLUDE_TWIDDLE_32_H__
+#define __INCLUDE_TWIDDLE_32_H__
 
 #include <stdint.h>
 
 #define FFT_SIZE_MAX	1024
 
 /* in Q1.31, generated from cos(i * 2 * pi / FFT_SIZE_MAX) */
-const int32_t twiddle_real[FFT_SIZE_MAX] = {
+const int32_t twiddle_real_32[FFT_SIZE_MAX] = {
```

Comment:
what's the input period for this MFCC? is it used for capture?

Reply:
The component adapts to (near) any input period. Currently the capture pipeline is scheduled 1 ms. It accumulates samples until the FFT can be computed. After that it computes the output every FFT hop. To have even processing load the pipeline should be (event) scheduled to match FFT hop.